src/main/scala/com/gd/twitterstreamingtToDatalake/transformTweets.scala

```diff
@@ -0,0 +1,11 @@
+package com.gd.twitterstreamingtToDatalake
+
+object transformTweets {
```

Comment:
please follow the Scala naming convention guidelines: https://docs.scala-lang.org/style/naming-conventions.html